tutorials/cookbooks/running_production.md

```diff
@@ -0,0 +1,308 @@
+  
+## A quick introduction
+  
+Hopefully, you already know that [MeiliSearch](https://github.com/meilisearch/MeiliSearch) is a powerful and fast search engine built in [Rust](https://www.rust-lang.org) as an Open Source tool. It was designed to provide users with a very useful and customizable search experience including features like typo-tolerance, filtering and synonyms out of the box, for any kind of project. Running a Meilisearch for testing purposes is incredibly easy, as [many alternatives](https://docs.meilisearch.com/guides/introduction/quick_start_guide.html) are available: Docker, brew, aptitude, binaries, a simple curl or even the source code. If you are new to MeiliSearch, we suggest that you make a tour arround our [Documentation](https://docs.meilisearch.com/)  
+  
+  
+Running MeiliSearch on your own machine for your weekend project is fun, let's agree on that. But we are here to **take you to the next step**. You probably want to go live, and deploy a project in production, take it to the real word. What are the steps and details you need to **deploy MeiliSearch in production** and be sure that it is **safe and ready to use**?  
+
+## Content of this article
+
+[Step 1: Install MeiliSearch](#Step-1-Install-MeiliSearch)
+
+[Step 2: Run MeiliSearch as a service](#Step-2-Run-MeiliSearch-as-a-service)
++ [2.1 Create a service file](#21-Create-a-service-file)
++ [2.2. Enable and start service](#22-Enable-and-start-service)
+
+[Step 3: Secure and finish your setup. Using a Reverse Proxy, domain name and HTTPS](#Step-3-Secure-and-finish-your-setup-Using-a-Reverse-Proxy-domain-name-and-HTTPS)
++ [3.1. Creating a Reverse Proxy with Nginx](#31-Creating-a-Reverse-Proxy-with-Nginx)
++ [3.2. Activating SSL (HTTPS) on your MeiliSearch](#32-Activating-SSL-HTTPS-on-your-MeiliSearch)
+
+[Conclusion](#Conclusion)
+  
+## Get your MeiliSearch ready for production
+  
+For this tutorial, we will be using a Debian 10 server, running on DigitalOcean. You can easily try it on your own, with plans starting at $5/month. And if you want some credits to start running your MeiliSearch and are not already registered on DigitalOcean, you can get $100 for free using [this referral link](https://m.do.co/c/7c67bd97e101).  
+
+
+
+## Prerequisites
+
+- An up-to-date server that runs Debian 10
+- An ssh keypair to connect to that machine
+
+> *TIPS:* learn how to connect via SSH to your [DigitalOcean droplet](https://www.digitalocean.com/docs/droplets/how-to/connect-with-ssh/) or any [Linux or windows server](https://phoenixnap.com/kb/ssh-to-connect-to-remote-server-linux-or-windows)
+
+## Step 1: Install MeiliSearch
+
+Installing and running MeiliSearch is an easy and straightforward process. In order to keep this tutorial as simple as possible, we are going to use a MeiliSearch installation script that will take care of the process for you. It will get a copy of a MeiliSearch binary in your machine and let you use it inmediatley.
```

Comment:
```suggestion
Installing and running MeiliSearch is easy and straightforward. In order to keep this tutorial as simple as possible, we will use a script that will carry out the installation process. It will copy a binary of MeiliSearch to your machine and enable you to use it immediately.
```